ci/meta.yml

```diff
@@ -29,7 +29,9 @@ meta:
             - -exc
             - |
               python bootstrap_terraform.py
+              sed -i '/^assume_role/ d' terraform.tfvars
```

Comment:
If we don't want this in here, would it not be better to use the `variable.assume_role` in variables.tf and override it, if running locally?